src/Gui/CommandView.cpp

```diff
@@ -2767,7 +2729,8 @@ void StdViewBoxZoom::activated(int iMsg)
     if ( view ) {
         View3DInventorViewer* viewer = view->getViewer();
         if (!viewer->isSelecting()) {
-            SelectionCallbackHandler::Create(viewer, View3DInventorViewer::BoxZoom, QCursor(QPixmap(cursor_box_zoom), 7, 7));
+            QPixmap px(Gui::BitmapFactory().pixmapFromSvg("zoom-border-cross", QSize(32, 32)));

```

Comment:
it looks pixelated on HiDPI screens.
Would it work with: `32 * devicePixelRatio()` and also for the `QCursor(px, 6 * devicePixelRatio(), 6 * devicePixelRatio())` ?

Reply:
What's probably needed is to pass the size of 32 * devicePixelRatio() and then set the device pixel ratio to the returned pixmap. The hotspot of the cursor only needs to be adjusted for some Linux platforms while on macOS and Windows it's done automatically.

That's the way how it's done e.g. in DrawSketchHandler::setSvgCursor